app/scripts/views/mixins/notifications-mixin.js

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A view mixin to interface with the notifications channel
+ *
+ * Any registered handlers are automatically unregistered
+ * when the view is destroyed.
+ */
+
+define([
+], function () {
+  'use strict';
+
+  var NotificationsChannelMixin = {
+    notificationsInitialize: function (options) {
+      options = options || {};
+
+      /**
+       * A list of handlers registered for this view is kept
+       * so all listeners can be unregistered when the view is
+       * destroyed.
+       */
+      this._notificationsMessages = [];
+      this._notifications = options.notifications;
+
+      for (var eventName in this.notifications) {
+        console.log('registering', eventName);
```

Comment:
I know this is only a WIP review but after failing to comment on the in `crosstab`, it would be remiss of me not to remind you to delete these before merging. :)